controllers/authController.js

```diff
@@ -4,42 +4,42 @@ const UserCredentials = require('../models/userCredentials');
 const mongoose = require('mongoose');
 
 // Sign-Up a new employee
-async function signUp(req, res){
-    try{
-        const {email, password} = req.body;
+async function signUp(req, res) {
+    try {
+        const { email, password } = req.body;
 
         // regex to validate email
         let emailRegex = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w\w+)+$/;
-        if(!emailRegex.test(email))throw Error("Invalid email");
+        if (!emailRegex.test(email)) throw Error("Invalid email");
 
         // check if existing user
-        const existingUser = await UserCredentials.findOne({email});
-        if(existingUser){
-            return res.status(409).json({ success: false, message: 'Email already exists'});
+        const existingUser = await UserCredentials.findOne({ email });
+        if (existingUser) {
+            return res.status(409).json({ success: false, message: 'Email already exists' });
         }
-        else{
```

Comment:
as there is return statement in if condition, no need to explicitly use else
By default the next part is else

Reply:
I thought this will also create confusion to find out where exactly this else is completing

Reply:
Oh..! Ok. Understood Ram! Thank you.